ui/StatusQ/src/StatusQ/Controls/StatusColorRadioButton.qml

```diff
@@ -24,6 +25,23 @@ RadioButton {
         border.width: 1
         border.color: Theme.palette.directColor7
 
+        Item {
+            id: dualColor
+            property string yinYangColor: Utils.getYinYangColor(radioButtonColor)
```

Comment:
Why is this a separate property on its own?

Reply:
used twice, but I'll move it under a QtObject instead

Reply:
also needed because I need to show yinyang  like this 
![image](https://github.com/status-im/status-desktop/assets/60327365/d0b3213c-8f0b-40f9-aa7f-3f9b371fd111)
I need to show black and white in case its yinyang and hence this property